survey/models/question.py

```diff
@@ -106,7 +106,7 @@ def get_clean_choices(self):
         if self.choices is None:
             return []
         choices_list = []
-        for choice in self.choices.split(","):
```

Comment:
This was actually a magic string, nice catch! Also, it permits to use comma in question by using an exotic separator and that was not possible before. Very nice change !

Reply:
So do you apply this? I'm not sure about my tests naming according to Codacy/PR Quality Review, where am I wrong here?

Reply:
oh just saw you main comment, thanks